test/Interop/Cxx/function/emit-called-function-ir.swift

```diff
@@ -0,0 +1,14 @@
+// This is a reproducer for a bug where Swift was not emitting code for a
```

Comment:
Could you rephrase this as a test for emitting inline functions that are used in general? The "regression test" wording is useful for obscure combinations, but this seems like a pretty basic case that we should be testing regardless.

Reply:
Good point, done.